tasks/prime.py

```diff
@@ -38,6 +38,35 @@ def fetch_mto_updates(self):
         else:
             logger.info(f"ℹ️ Num MTOs returned: {len(json_body)}")
 
+    @tag("mtoServiceItem", "createMTOServiceItem")
+    @task
+    def create_mto_service_item(self):
+        payload = {
+            "moveTaskOrderID": "5d4b25bb-eb04-4c03-9a81-ee0398cb779e",
+            "mtoShipmentID": "475579d5-aaa4-4755-8c43-c510381ff9b5",
+            "modelType": "MTOServiceItemDDFSIT",
+            "reServiceID": "8d600f25-1def-422d-b159-617c7d59156e",
```

Comment:
These values - `moveTaskOrderID`, `mtoShipmentID` and `reServiceID` should be dynamic values. This will be addressed as apart of an ongoing [effort to more easily generate fake data](https://dp3.atlassian.net/secure/RapidBoard.jspa?rapidView=25&projectKey=MB&modal=detail&selectedIssue=MB-2968).